Sources/SwiftBSON/BSONDocumentIterator.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 import NIO
 
+/// :nodoc:
```

Comment:
anything not listed in a custom category automatically gets put into an "other" section, so this was necessary to keep this type from showing up there.